src/SurfaceFluxes.jl

```diff
@@ -571,16 +584,16 @@ function heat_exchange_coefficient(
 )
     thermo_params = SFP.thermodynamics_params(param_set)
     FT = eltype(L_MO)
-    Δθ = TD.dry_pottemp(thermo_params, ts_in(sc)) - TD.dry_pottemp(thermo_params, ts_sfc(sc))
+    transport = UF.HeatTransport()
     ustar = compute_ustar(param_set, L_MO, sc, uft, scheme)
-    θstar = Δθ * compute_physical_scale_coeff(param_set, sc, L_MO, UF.HeatTransport(), uft, scheme)
-    ρ_in = TD.air_density(thermo_params, ts_in(sc))
-    ρ_sfc = TD.air_density(thermo_params, ts_sfc(sc))
-    Δρ = ρ_in - ρ_sfc
-    Ch = if Δθ ≈ FT(0) || windspeed(sc) ≈ FT(0) || Δρ ≈ FT(0)
-        FT(0)
+    DSEᵥ_sfc = compute_DSEᵥ(param_set, sc, ts_sfc(sc), z_sfc(sc))
+    DSEᵥ_in = compute_DSEᵥ(param_set, sc, ts_in(sc), z_in(sc))
+    ΔDSEᵥ = DSEᵥ_in - DSEᵥ_sfc
+    ϕ_heat = compute_physical_scale_coeff(param_set, sc, L_MO, transport, uft, scheme)
+    Ch = if ΔDSEᵥ ≈ FT(0)
+        Ch = (κ / log(Δz(sc) / z0(sc, transport)))^2
```

Comment:
@tapios wants to double check this line I think.

Reply:
Yes, please make sure this is the correct neutral limit of your similarity functions.